src/containers/add-attachments/AddAttachmets.tsx

```diff
@@ -0,0 +1,108 @@
+import { ChangeEvent, FC, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { useModalContext } from '~/context/modal-context'
+import { ajustColumns } from '~/utils/helper-functions'
+import useBreakpoints from '~/hooks/use-breakpoints'
+import useSort from '~/hooks/table/use-sort'
+
+import { Box } from '@mui/material'
+import Typography from '@mui/material/Typography'
+import SearchIcon from '@mui/icons-material/Search'
```

Comment:
mui libs should be above after react imports. Also, it is better to use import Box from '@mui/material/Box' because it reduces bundle size

Reply:
done